util/conversion/conversion.go

```diff
@@ -128,7 +129,26 @@ func UnmarshalData(from metav1.Object, to interface{}) (bool, error) {
 
 // GetFuzzer returns a new fuzzer to be used for testing.
 func GetFuzzer(scheme *runtime.Scheme, funcs ...fuzzer.FuzzerFuncs) *fuzz.Fuzzer {
-	funcs = append([]fuzzer.FuzzerFuncs{metafuzzer.Funcs}, funcs...)
+	funcs = append([]fuzzer.FuzzerFuncs{
+		metafuzzer.Funcs,
+		func(_ runtimeserializer.CodecFactory) []interface{} {
+			return []interface{}{
+				// Custom fuzzer for metav1.Time pointers which weren't
+				// fuzzed and always resulted in `nil` values.
+				// This implementation is somewhat similar to the one provided
+				// in the metafuzzer.Funcs.
+				func(input *metav1.Time, c fuzz.Continue) {
+					if input != nil {
+						var sec, nsec uint32
```

Comment:
Is there a reson to not use `int64` directly here?

Like in:
```go
			var sec, nsec int64
			c.Fuzz(&sec)
			c.Fuzz(&nsec)
			j.CreationTimestamp = metav1.Unix(sec, nsec).Rfc3339Copy()
```

Reply:
int64, being large and allowing negative numbers generates invalid years when performing the rfc339 copy

Reply:
Makes sense.